addon/TabTracker.js

```diff
@@ -43,6 +40,12 @@ TabTracker.prototype = {
     return this._tabData;
   },
 
+  init(trackableURLs, placesQueries, experimentId) {
```

Comment:
Do we expect to always have `trackableURLs` and sometimes the others (in the context of tracking existing tiles)? If so, it might be better to place the optional stuff in an options object to avoid worrying about ordering of optional parameters. We can do this in a followup once we're more certain about which pings we want for sure guiding us to what parameters are required. Please file and link back to this issue.